### PR TITLE
Features/populate organization fields 114

### PR DESCRIPTION
**Pull request by djarredon**

### CODE REVIEW

File: client/src/components/organization/index.js [hunk 1]
```diff
@@ -22,75 +22,136 @@ import { orgPage } from '../../constants/viewStyles'
...
+      id,
+      firstName,
+      lastName,
+      mainTitle,
+      media
+    } = employee
+
+    var imageUrl = null;
+
+    if (media && media[0])
```
Comment (mballeza) [inline client/src/components/organization/index.js]: Use the same "reducedMedia" function used in the organization reducer to get the url of the image. I'm not sure if it's safe to assume that the image is always in index 0.

**Review by mballeza — CHANGES_REQUESTED** [review]
Just the one review comment in addition to the reviews I made for the professional branch. The employee table looks good. If you want to see an org with at least one employee, try an ID of 34 or 36.

**Review by mballeza — CHANGES_REQUESTED** [review]
Looks great, see the review from the professional branch, then I'll approve.

**Review by iandusenbury — APPROVED** [review]

**Review by mballeza — CHANGES_REQUESTED** [review]
Has some linting errors still in components/organization and the reducer. I'm getting some errors in the console related to the video player, though I'm not sure if it has to be addressed right now. Other than that, the page itself looks great! Just needs a merge with the current dev branch and it'll be good to go.

**Review by mballeza — APPROVED** [review]

**Pull request merged**